samples/voting/README.md

```diff
@@ -9,11 +9,44 @@ The project should be immediately runnable by calling `tye run` from the directo
 
 A few things need to be configured before deploying to Kubernetes.
 
-- Setting up Redis. A connection string needs to be provided to connect to Redis. You can follow our tutorial on [setting up redis in your cluster](../../docs/tutorials/hello-tye/02_add_redis.md).
-- Setting up postgresql. A connection string eventually needs to be provided to tye for postgresql.
-- Deploying the ingress.yaml by calling:
+- [Create an Azure container registry using the Azure portal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal) or you can use your public [docker hub account](https://hub.docker.com/)
 
+- `Redis` can be deployed using the below command :
+
+    ```powershell
```

Comment:
Why powershell?

Reply:
Hi @davidfowl , 
Yes, instead of marking it as specific shell. It would be better to use only "triple backticks" to show the command text. I have made the changes and updated the PR for your further review.

Thanks.